bee-message/bee-byte-cost/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "bee-byte-cost"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2021"
+description = "Defines the traits for computing the byte cost."
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = [ "iota", "tangle", "bee", "framework", "bytecost", "dust" ]
```

Comment:
wasn't that limited to at most 5 keywords? :thinking:

Reply:
Yes we can't release the crate if it has more than 5 keywords